geps/gep-1731/metadata.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: internal.gateway.networking.k8s.io/v1alpha1
+kind: GEPDetails
+number: 1731
+name: Configurable Retries
```

Comment:
Tiny Nit: Recommend calling this "HTTPRoute Retries" (same applies elsewhere)

Reply:
Makes sense, I started with the more generic name first in case it would have made more sense to implement as policy attachment or another alternative.